modules/retail/pages/retail-formulas-intro.adoc

```diff
@@ -15,12 +15,15 @@ Branch servers are configured using formulas.
 Formulas can be configured using {productname} {webui}, or the {productname} XMLRPC API.
 To fully configure {productname} {smr}, these formulas need to be enabled and configured on the branch server:
 
-* Branch network formula, see xref:specialized-guides:salt/salt-formula-branchnetwork.adoc[]
 * Bind formula, see xref:specialized-guides:salt/salt-formula-bind.adoc[]
 * DHCPD formula, see xref:specialized-guides:salt/salt-formula-dhcpd.adoc[]
+* VSFTP formula, see xref:specialized-guides:salt/salt-formula-vsftpd.adoc[]
+
+In addition, the following formulas can be used for connecting {productname} 5.x with {productname} 4.3.:
```

Comment:
This should be at the top of this page, because these formulas can't be used with 5.0 proxy/rbs.

VSFTP formula should be removed from the retail documentation completely, as we do not use that anymore even in 4.3.

Bind and DHCPd formulas can be used, but not on the branch server because SLE Micro does not have bind and dhcpd in the repositories.